src/components/card/card.scss

```diff
@@ -464,3 +483,60 @@
     }
   }
 }
+
+
+// set padding if background color is different from card color
+[class*="bg-"] [class*="bg-"] {
+  .card__body {
+    padding: 2rem;
+  }
+}
+
+// unset matching background color padding
+.bg--gold .bg--gold.card--stacked:not(.card--enclosed),
+.bg--white .bg--white.card--stacked:not(.card--enclosed),
+.bg--black .bg--black.card--stacked:not(.card--enclosed),
+.bg--gray .bg--gray.card--stacked:not(.card--enclosed) {
+  .card__body {
+    padding: 2rem 0;
+  }
+}
+
+// background inherit overrides
+.card.card--enclosed {
+
+  &.bg--white,
+  &.bg-pattern--brain,
+  &.bg--gold .bg--white,
+  &.bg--gold .bg-pattern--brain,
+  &.bg-pattern--brain--gold .bg--white,
+  &.bg-pattern--brain--gold .bg-pattern--brain,
+  .bg--white &,
+  .bg-pattern--brain &,
+  .bg--gold .bg--white &,
+  .bg--gold .bg-pattern--brain &,
+  .bg-pattern--brain--gold .bg--white &,
+  .bg-pattern--brain--gold .bg-pattern--brain & {
+    border: 1px solid rgba(0, 0, 0, 0.125);
+  }
+}
+
+.card {
+
+  &.bg--white,
+  &.bg-pattern--brain,
+  &.bg--gold .bg--white,
+  &.bg--gold .bg-pattern--brain,
+  &.bg-pattern--brain--gold .bg--white,
+  &.bg-pattern--brain--gold .bg-pattern--brain,
```

Comment:
I don't understand the use of this unless you think it is possible that we are going to allow background colors inside of cards - in addition to the card having a background color itself.